@here/harp-mapview-decoder/lib/GeoJsonTiler.ts

```diff
@@ -7,10 +7,19 @@
 import { GeoJson, ITiler } from "@here/harp-datasource-protocol";
 import { TileKey } from "@here/harp-geoutils";
 
-// tslint:disable-next-line:no-var-requires
-const geojsonvtExport = require("geojson-vt");
+// @ts-ignore
+import * as geojsonvtExport from "geojson-vt";
+
 // to be able to run tests on nodejs
-const geojsonvt = geojsonvtExport.default || geojsonvtExport;
+const geojsonvt = geojsonvtExport.default ?? geojsonvtExport;
+
+const EXTENT = 4096;
+
+// the factor used to compute the size of the buffer.
+const BUFFER_FACTOR = 0.05;
+
+// tslint:disable-next-line: no-bitwise
+const BUFFER = -(-Math.ceil(EXTENT * BUFFER_FACTOR) & -2);
```

Comment:
what does -2 mean?

Reply:
yes, it's a round up with alignment of 2.

Reply:
```
> const RoundUp = (x,align=2) => -((-Math.ceil(x)) & -align)
undefined
> RoundUp(4 * 1024 * 0.05)
206
```